CONTRIBUTING_DE.md

```diff
@@ -0,0 +1,180 @@
+- [x] **Lint:** :wrench: Stelle sicher, dass Linters auf alle neuen oder geänderten Dateien angewendet wurden.
+- [x] **Bedeutsame Änderungen:** :mag_right: Bestätige, dass jede geänderte Datei einen sinnvollen Inhalt beiträgt und vermeide unbedeutende Änderungen wie reine Anpassungen von Leerzeichen.
+- [x] **Testing:** :test_tube: Schreibe Tests für alle neuen Funktionen und aktualisiere bestehende Tests, wenn du Änderungen an Funktionalitäten vorgenommen hast.
+- [x] **Sprache & Rechtschreibung:** :book: Verwende Englisch für alle Variablennamen, Klassennamen, Funktionen, Kommentare usw., und stelle sicher, dass aller hinzugefügter Inhalt Rechtschreibprüfungen durchlaufen hat.
+- [x] **Sensible Informationen:** :no_entry: Überprüfe vor dem Einreichen noch einmal, um sicherzustellen, dass keine Passwörter, Zugangsdaten oder lokale Konfigurationen in deinen Änderungen vorhanden sind.
+- [x] **Kontinuierliche Integration:** :green_circle: Stelle sicher, dass der GitHub Actions CI-Build erfolgreich abgeschlossen werden kann, ohne Testfehler.
+
+## Datenbank :floppy_disk:
+
+### Verwende Dev-Data für die Lokale Entwicklung :construction_worker:
+Um die Entwicklung zu vereinfachen und Konsistenz in lokalen Umgebungen zu gewährleisten,
+bieten wir einen vordefinierten Datensatz an, der als 'Dev-Daten' bekannt ist.
+Dieser Datensatz ist darauf ausgerichtet, den Testprozess zu optimieren und eine diverse
+Datenabbildung zu erlangen um auch Randfälle zu Testen.
+
+### Empfohlener Testbenutzer :bust_in_silhouette:
+Um dich in Entwicklungsumgebungen wie [localhost:3000](http://localhost:3000) einzuloggen, verwende die Benutzerdaten `test@example.com / test`.
+Dieser Benutzer wurde mit einer umfassenden Sammlung von Lagern ausgestattet, die für das Testen der meisten Funktionen und Szenarien ausreichen sollten.
+
+### Rückmeldungen zu den Testdaten :loudspeaker:
+Wir bemühen uns ständig, unsere 'Dev-Daten' zu verbessern.
+Wenn du Lücken erkennst oder glaubst, dass es ein zusätzliches Szenario abdecken sollte,
+öffne bitte ein Issue, um uns darüber zu informieren.
+
+### Dokumentation  :mag:
+Für ein tieferes Verständnis der 'Dev-Daten' haben wir dieses [README](./api/migrations/dev-data/README.md) (nur auf Englisch verfügbar) erstellt.
+
+### Konsistentes Testen in verschiedenen Umgebungen :globe_with_meridians:
+Die 'Dev-Daten' werden in allen Entwicklungsumgebungen repliziert.
+Wir empfehlen deren Verwendung für konsistente Tests.
+Wenn du ein Problem oder einen Fehler meldest: Referenziere doch ein spezifisches Beispiel aus den 'Dev-Daten'.
+Da die Daten, einschließlich der IDs, konstant bleiben, kann jeder das von dir hervorgehobene Verhalten leicht nachvollziehen und reproduzieren.
+
+## Discord :speech_balloon:
+Wir verstehen, dass das Einrichten einer Entwicklungsumgebung manchmal knifflig sein kann,
+besonders bei unterschiedlichen Systemen und Konfigurationen.
+Wenn du auf Probleme stösst oder Hindernisse während der Einrichtung begegnest,
```

Comment:
Hindernissen mit n am Schluss

Reply:
Ich bin mir nicht sicher, ich habe [Duden Mentor](https://mentor.duden.de/) und ChatGPT gefragt.
![image](https://github.com/ecamp/ecamp3/assets/57986114/0b4d885a-6cbe-4ec6-8c8c-b99a558fe142)

Bei jedem durchlesen des Satzes werde ich unsicherer.

Da Duuden Mentor mit 'n' auch keinen Fehler findet habe ich es in 'feedback fix XI' hinzugefügt.
Vielleicht kann mich da ja jemand erleuchten.

Reply:
Ich habe 3 Kolleg\*innen befragt: Eine Studentin welche deutsche Sprach- und Literaturwissenschaften studiert, ein Gymnasiallehrer der Deutsch und Geschichte unterrichtet und eine Neulatein-Studentin welche auch schon Styleguides für deutschsprachige Bücher und Artikel verfasst hat. Die Antworten:

> Würds i dem Satz au mit 'n' schriebe... Wenns 'auf Hindernisse oder Probleme stösst' heisse würd, dänn ohni aber mit begegnest mit...

> Mit n hätti gseit, begegnen verlangt dativ bim Objekt

> Mit n und d Satzstellig ändere: „oder während der Einrichtung Hindernissen begegnest“